spec/requests/api/v1/user_can_randomise_a_movie_spec.rb

```diff
@@ -0,0 +1,19 @@
+RSpec.describe 'Get /api/v1/movies', type: :request do
+  describe 'successfully' do
+    before do 
+      get '/api/v1/movies/random'
+    end
+
+    it 'is expected to get a 200 response' do
+      expect(response).to have_http_status 200
+    end
+
+    it 'is expected to return a movie' do
+      expect(response_json["movie"]).to include("title")
+    end
+
```

Comment:
When you correct the functionality in the controller you can change the response in the fixture to be one movie and then you can test more specific for that movie.